keps/sig-apps/20181220-statefulset-volume-expansion.md

```diff
@@ -0,0 +1,93 @@
+---
+title: StatefulSet Volume Expansion
+authors:
+  - "@sidakm"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-storage
+reviewers:
+  - "@janetkuo"
+  - "@gnufied"
+approvers:
+  - "@kow3ns" 
+editor: TBD
+creation-date: 2018-12-20
+last-updated: 2019-02-04
+status: provisional
+see-also:
```

Comment:
Can you add existing PRs/design for online and volume expansion proposal here?

Reply:
Are these links sufficient?
For Online Expansion: 
https://github.com/kubernetes/enhancements/issues/531
https://github.com/kubernetes/enhancements/pull/737

For original expansion:
https://github.com/kubernetes/enhancements/issues/284

Reply:
yeah should be good enough. thanks